README.md

```diff
@@ -7,6 +7,30 @@ or dynamic objects, and articulated robots with various sensors, and acutators.
 The format of SDFormat is also described by XML, which facilitates updates and
 allows conversion from previous versions.
 
+## Documentation
+
+See the [SDFormat Website](http://sdformat.org/) for:
+
+* [An outline of the SDFormat XML specification](http://sdformat.org/spec)
+* [Documentation on specification specifics](http://sdformat.org/tutorials?cat=specification)
+* [Proposals for modifications](http://sdformat.org/tutorials?cat=pose_semantics_docs)
+* [Developer instructions](http://sdformat.org/tutorials?tut=roadmap)
+  (e.g. installation and a roadmap)
+
+For the source of this documentation, please see the
+[`sdf_tutorials`](https://github.com/osrf/sdf_tutorials) repository.
+
+<!--
+TODO(eric.cousineau): Add link to sdf_web_app once it's public.
+TODO(eric.cousineau): Add a link to API once later versions work.
+TODO(eric.cousineau): Add a link to Downloads once they're easier to navigate.
+TODO(eric.cousineau): Fix proposals to not use `pose_semantics_docs` as its
+name?
+TODO(eric.cousineau): Move installation instructions to sdf_tutorials, and link
+there?
+TODO(eric.cousineau): Move terminology section to sdf_tutorials?
```

Comment:
The last 3 sound reasonable to me, but maybe these TODOs should be point to issues on https://github.com/osrf/sdf_tutorials/ ?

Reply:
Done.

Reply:
it sounds like she was asking for a link to issues in the TODO comments?

ie: https://github.com/osrf/sdf_tutorials/issues/1

I don't think it hurts to have the terminology in both places. So I wouldn't "Move terminology section to sdf_tutorials" but rather copy it there. Duplicate info isn't great, it's short enough, and I think it's helpful in both places